push_notifications/migrations/0010_alter_gcmdevice_cloud_message_type.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.2.10 on 2024-04-12 16:29
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("push_notifications", "0009_alter_apnsdevice_device_id"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="gcmdevice",
+            name="cloud_message_type",
+            field=models.CharField(
+                choices=[
+                    ("FCM", "Firebase Cloud Message"),
+                    ("GCM", "Google Cloud Message"),
+                ],
+                default="FCM",
```

Comment:
actually, this line is the main reason for a callable default. otherwise the migration would be hard coded to `"FCM"` and projects using a different default in their settings would complain about a missing migration.